css/header.css

```diff
@@ -9,7 +9,7 @@
   align-items: center;
   justify-content: space-between;
   background-color: #FFF;
-  z-index: 1037;
+  z-index: 997;
```

Comment:
Hey sagar, made this change so header could always stay above projects/themes when scrolling.

Reply:
It will still be above them, i just made this value less than the loader overlay value